server/main.lua

```diff
@@ -48,7 +48,8 @@ RegisterNetEvent('qb-apartments:returnBucket', function()
 end)
 
 RegisterNetEvent('apartments:server:CreateApartment', function(type, label)
-    local player = QBCore.Functions.GetPlayer(source)
+    local src = source
```

Comment:
We don't yield in this event before using source so saving is not necessary

Reply:
Will update in the future as it's done in most events either way

Reply:
I'm not sure why, but when triggering client events at the bottom of this event, was getting a native exception that first argument was null. using local src fixes this exception.